webview-ui/src/CreateCluster/CreateClusterInput.tsx

```diff
@@ -137,21 +139,21 @@ export function CreateClusterInput(props: CreateClusterInputProps) {
                         </span>
                     )}
 
-                    {newResourceGroup && (
+                    {/* {newResourceGroup && (
                         <>
                             <VSCodeTextField
                                 readOnly
                                 className={styles.midControl}
-                                value={`${newResourceGroup.name} (${newResourceGroup.location})`}
+                                value={`(New) ${newResourceGroup.name} (${newResourceGroup.location})`}
                             ></VSCodeTextField>
                             <VSCodeButton className={styles.sideControl} onClick={() => setNewResourceGroup(null)}>
                                 Clear
                             </VSCodeButton>
                         </>
-                    )}
+                    )} */}
 
                     <label htmlFor="name-input" className={styles.label}>
```

Comment:
After discussion with the Portal team, I think we agreed to provide a Location drop-down, didn't we? Is this planned for a separate PR? If so, we should probably create a work item for it.

Reply:
@peterbom Yes even @qpetraroia brought this up. But if you see the resource groups are tied to location and cluster would be created in that location. Do we need separate location drop down again? Portal has them all in one single page both creation/selection of resource groups. In vscode we have them as separate dialog. Please let me know. 

![image](https://github.com/Azure/vscode-aks-tools/assets/105889062/a0c4dbbb-81f0-4613-be35-7bf715740f5e)